_authors/osci_2023/noahdohrmann.markdown

```diff
@@ -0,0 +1,7 @@
+---
+name: Noah Dohrmann
+linkedin: noah-dohrmann
+github: n-dohrmann
+---
+
+**<Noah Dohrmann>** Hi! Happy to get started here!
```

Comment:
One last request, can you remove the angled brackets <> around your name on this line?

Reply:
Yep! Sorry, I was confused about the syntax there. Have a good evening!